git_machete/tests/functional/test_machete.py

```diff
@@ -26,6 +29,148 @@ def get_head_commit_hash() -> str:
     return os.popen("git rev-parse HEAD").read().strip()
 
 
+class FakeCommandLineOptions(CommandLineOptions):
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
Whoops so without explicit call to superclass c'tor... it just won't be called by default? or an error will be thrown?

Reply:
Without `super()` it won't be called. It has to be told explicitly to python to call it.